POSTINSTALL.md

```diff
@@ -136,12 +136,12 @@ To reindex data, use the steps listed above for "Backfill or import existing doc
 
 The TLDR for this section is:
 
-- Field names will be lowercased before being indexed, so the field name in app search may differ from the field name in Firestore.
 - `text` and `number` type fields are indexed as-is to App Search.
 - `geo` and `timestamp` fields are formatted slightly differently when indexed.
 - `map`, `boolean`, and `reference` are not supported by App Search, they will be indexed as text.
 - nested arrays are not supported at all and will be dropped before indexing in App Search.
-- While `map`s are not supported, you _can_ specify that fields within a map get indexed as top level fields in App Search.
+- While `map`s are not supported, you _can_ specify that fields within a map get indexed as top level fields in App Search, using the `__` syntax when configuring indexed fields
```

Comment:
Is using `__` over `.` to lower conflict risk with an actual field name that uses `.`?  Admittedly this syntax does feel a little unnatural.

Reply:
I agree it feels a bit unnatural. It's to reduce the chance of conflicts *and* the name of the indexed field ends up matching the name of the field that actually gets indexed... one less point of confusion. Meaning, when you specify `field__subfield` it will actually be indexed as `field__subfield`